src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -289,35 +303,19 @@ private boolean runRepair() {
       }
 
       try (Timer.Context cxt1 = context.metricRegistry.timer(metricNameForRepairing(segment)).time()) {
-        boolean segmentsLocked = false;
+        //boolean segmentsLocked = false;
         try {
           LOG.debug("Enter synchronized section with segment ID {}", segmentId);
           synchronized (condition) {
-            if (!(segmentsLocked = lockSegmentRunners())) {
+            /*if (!(segmentsLocked = lockSegmentRunners())) {
               // XXX – not expected to happen, STARTED run state should be "good" (opportunistic) enough
               LOG.warn(
                   "Cannot run segment {} as another Reaper holds the lock on repair run {}. Will try again later",
                   segmentId,
                   segment.getRunId());
 
               return false;
-            }
-
-            // ~double-checking idiom, only applies to non-incremental and distributed storage
-            if (!repairUnit.getIncrementalRepair() && context.storage instanceof IDistributedStorage) {
-              Map<String, String> dcByNode = segment.getReplicas();
-              if (isRepairRunningOnNodes(segment, dcByNode, keyspace, cluster)) {
-                LOG.warn(
-                    "Post-lock, cannot run segment {} for repair {} at the moment. Will try again later",
-                    segmentId,
-                    segment.getRunId());
-
-                try {
-                  Thread.sleep(SLEEP_TIME_AFTER_POSTPONE_IN_MS);
-                } catch (InterruptedException ignore) { }
-                return false;
-              }
-            }
+            }*/
```

Comment:
the commented stuff in this method can be removed.

Reply:
Yes, done in the next PR: https://github.com/thelastpickle/cassandra-reaper/pull/996/files#diff-2e6b3c333eecade8a9d85f64416757e3edd4bbf4f1b67f9ebd60d7dd780f899aL310-L318